profiles/josevale99.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/39961893/213903888-4347babe-71b8-41b3-b69e-836c1fe87503.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/213935618-d8bbd295-4fee-45cb-94d1-5278624b9042.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/213935618-d8bbd295-4fee-45cb-94d1-5278624b9042.png",
```